test/jasmine/karma.conf.js

```diff
@@ -1,30 +1,102 @@
 /* eslint-env node*/
 
-// Karma configuration
-
-/*
- * Test file globs can be passed with an argument.
- *
- * Example:
- *
- *  $ npm run test-jasmine -- tests/axes_test.js
- *
- * will only run the tests in axes_test.js
- *
- */
-
+var path = require('path');
+var minimist = require('minimist');
 var constants = require('../../tasks/util/constants');
 
-var arg = process.argv[4];
-
 var isCI = !!process.env.CIRCLECI;
-var testFileGlob = arg ? arg : 'tests/*_test.js';
-var isSingleSuiteRun = (arg && arg.indexOf('bundle_tests/') === -1);
-var isRequireJSTest = (arg && arg.indexOf('bundle_tests/requirejs') !== -1);
-var isIE9Test = (arg && arg.indexOf('bundle_tests/ie9') !== -1);
+var argv = minimist(process.argv.slice(4), {
+    string: ['bundleTest', 'width', 'height'],
+    'boolean': ['info', 'watch', 'verbose', 'Chrome', 'Firefox'],
+    alias: {
+        'Chrome': 'chrome',
+        'Firefox': ['firefox', 'FF'],
+        'bundleTest': ['bundletest', 'bundle_test']
+    },
+    'default': {
+        info: false,
+        watch: false,
```

Comment:
`watch` is set to `false` by default so that `npm run test-jasmine` exits on complete by default. 

We could alternatively make `npm run test-jasmine` auto-watch and auto-rerun like previously and a `--no-watch` flag instead. I'm open to suggestions